client/blocks/plan-thank-you-card/style.scss

```diff
@@ -0,0 +1,226 @@
+.plan-thank-you-card {
+	position: relative;
+	width: 100%;
+	max-width: 500px;
+	background-color: darken( $alert-green, 10 );
+	border-radius: 8px 8px 6px 6px;
+	overflow: hidden;
+	margin: 56px auto 24px;
+	color: $white;
+	font-family: $sans;
+	text-align: center;
+
+	@media ( max-width: 500px ) {
+		margin-top: 0;
+	}
+}
+
+.plan-thank-you-card__header {
+	background-color: $alert-green;
+	padding: 75px 0 30px;
+	position: relative;
+	z-index: 1;
```

Comment:
To be honest, I haven't yet had to use `z-index` :D. However, I know that we have a custom functionality for that. See https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/css.md#z-index please.